test/test_charge_drift_models.jl

```diff
@@ -82,41 +82,109 @@ end
 end
 
 @timed_testset "Charge Trapping: ConstantLifetimeChargeTrappingModel" begin
+    # test 1: parse the lifetimes and the inactive layer geometry
+    config_dict = SolidStateDetectors.parse_config_file(SSD_examples[:TrueCoaxial])
+    simA = @test_nowarn Simulation{T}(config_dict)
```

Comment:
You don't seem to use `config_dict` anywhere else, so you could just do
```suggestion
    simA = @test_nowarn Simulation{T}(SSD_examples[:TrueCoaxial])
```